### DISCUSSION TITLE
Start browser with sandbox enabled

### DISCUSSION THREAD

**Discussion by byt3bl33d3r** [browserless/chrome#2478] [Q&A] [answered]
Heya,

First off, love the project. Using the docker container, I'm trying to start chrome with sanboxing enabled. From the logs is seems like no matter what I try it keeps launching Chrome with `--no-sandbox` flag. I tried setting the `DEFAULT_IGNORE_DEFAULT_ARGS=["--no-sandbox"]` environment variable or setting it to `true`, seems like the flag is always passed. 

Haven't found any documentation on how to start Chrome without this flag either. What am i missing?

Thanks!

**Comment by joelgriffith** [comment, marked as answer]
Sadly we do inject it as we don't come bundled with an SECCOMP profile. We should look to add support in the future since it's a common pitfall for folks...
